### PR TITLE
Update tg.js 

### PR DESCRIPTION
**Pull request by callmedandann**

### CODE REVIEW

**Review by CaptainJay4201 — COMMENTED** [review]
924 - Translate "temporary ban" to "pansamantalang ipinagbabawal"
927 - Translate "switch" to "palit"
930 - Translate "Missing" to "Nawawalang"
934 - Translate "round" to "laban"
935 - Translate "Switching Sides" to "Magpalit ng Posisyon"
969 - Translate "1 round" to "isang laban"
971 - Change "I-try again" to "Ulitin"

**Comment by callmedandann** [comment]
> 924 - Translate "temporary ban" to "pansamantalang ipinagbabawal" 927 - Translate "switch" to "palit" 930 - Translate "Missing" to "Nawawalang" 934 - Translate "round" to "laban" 935 - Translate "Switching Sides" to "Magpalit ng Posisyon" 969 - Translate "1 round" to "isang laban" 971 - Change "I-try again" to "Ulitin"

Done, thanks for the suggestions.

File: tg.js [hunk 1]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
 chall.name.bzk Manira ng killstreak ng {0} beses {1}
 chall.name.cc Ikaw ay muntikang mamatay {0} na beses {1}
 chall.name.rk Makamit ang {0} na paghiganti
-event.doubleXP 2x Class XP enabled for this match
-ranked.join Joined Ranked Match. Abandoning a match will result in a temporary ban
-ranked.progress Game in Progress
```
Comment (Markzt) [inline tg.js:925]: 925 kasalukuyang nasa laro

File: tg.js [hunk 2]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
...
-ranked.team.spawn Please join a Team to Spawn
-ranked.class.spawn Please select a Class to Spawn
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
-modes.round.switch Switching Sides. Prepare for the next Round
-modes.trade.success Trade Successful
+event.doubleXP 2x Class XP ay naka-enabled para sa larong ito
+ranked.join Naka-join na sa Ranked match. Ang pag-abandona ay mag reresulta sa temporary ban. 
+ranked.progress Kasalukuyang naglalaro
+ranked.team Pumili muna ng team
```
Comment (Markzt) [inline tg.js:926]: 926 pumili muna ng grupo

File: tg.js [hunk 3]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
...
+ranked.switch Hindi makapag makapagpalit habang naglalaro
+ranked.spectate Naka-disable ang panonood sa ranked
+ranked.login Mag-login upang maka-spawn
+ranked.sync Nawawala ang Sync Data
+ranked.team.spawn Pumili muna ng team para maka-spawn
+ranked.class.spawn Pumili muna ng Klas para maka-spawn
+modes.round.win Na-secure na ng team mo ang round na ito
```
Comment (Markzt) [inline tg.js:933]: Na-secure na ng grupo mo and laban na ito

File: tg.js [hunk 4]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
...
+modes.round.loss Hindi na maipapanalo ang laban na ito.
+modes.round.switch Magpapalit ng posisyon. Mag-prepare para sa susunod na round.
```
Comment (Markzt) [inline tg.js:935]: 935 Magpapalit ng posisyon. Mag Handa Para sa susunod na laban

File: tg.js [hunk 5]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
...
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
-modes.round.switch Switching Sides. Prepare for the next Round
-modes.trade.success Trade Successful
+event.doubleXP 2x Class XP ay naka-enabled para sa larong ito
+ranked.join Naka-join na sa Ranked match. Ang pag-abandona ay mag reresulta sa temporary ban. 
+ranked.progress Kasalukuyang naglalaro
+ranked.team Pumili muna ng team
+ranked.switch Hindi makapag makapagpalit habang naglalaro
+ranked.spectate Naka-disable ang panonood sa ranked
```
Comment (Markzt) [inline tg.js:928]: 928 Naka-disable and pag-spectate sa ranked

File: tg.js [hunk 6]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
...
-ranked.class.spawn Please select a Class to Spawn
-modes.round.win Your Team has secured this round
-modes.round.loss This round is no longer winnable
-modes.round.switch Switching Sides. Prepare for the next Round
-modes.trade.success Trade Successful
+event.doubleXP 2x Class XP ay naka-enabled para sa larong ito
+ranked.join Naka-join na sa Ranked match. Ang pag-abandona ay mag reresulta sa temporary ban. 
+ranked.progress Kasalukuyang naglalaro
+ranked.team Pumili muna ng team
+ranked.switch Hindi makapag makapagpalit habang naglalaro
```
Comment (Markzt) [inline tg.js:927]: 927 Hindi Pwede mag palit habang naglalaro

File: tg.js [hunk 7]
```diff
@@ -920,20 +920,20 @@ chall.name.dmg Tumira ng {0} na damage {1}
...
+ranked.spectate Naka-disable ang panonood sa ranked
+ranked.login Mag-login upang maka-spawn
+ranked.sync Nawawala ang Sync Data
+ranked.team.spawn Pumili muna ng team para maka-spawn
```
Comment (Markzt) [inline tg.js:931]: 931 Pumili muna ng grupo para makapag spawn

**Review by Markzt — COMMENTED** [review]
i hope it help

File: tg.js (hunk 1 above)
Comment (Markzt) [reply tg.js:925]: sa kabila to kaso nag kamali ng review

File: tg.js [hunk 8]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
-commands.forfeit.ended Forfeiting Game
-commands.forfeit.ended.other Other Team Forfeited
+commands.forfeit.next Kailangan maglaro muna ng 1 laban
+commands.forfeit.wait Maghintay munang magsimula ang laro para maka-forfeit 
+commands.forfeit.later Ulitin muli sa ilang segundo 
+commands.forfeit.voted Ikaw ay naka vote na.
+commands.forfeit.limit Isang beses kalang makakaboto sa isang laro
+commands.forfeit.ended Fino-forfeit na ang laro
+commands.forfeit.ended.other Nag-forfeit na ang kabilang team
 server.message.join {0} joined the game
```
Comment (Markzt) [inline tg.js:976]: 976 {0} sumali sa laro

File: tg.js [hunk 9]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
 server.message.left {0} left the game
```
Comment (Markzt) [inline tg.js:977]: 977 {0} umalis sa laro

File: tg.js [hunk 10]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
 server.message.kicked {0} was kicked
```
Comment (Markzt) [inline tg.js:978]: 978 {0} ay napatalsik

File: tg.js [hunk 11]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
-commands.spraybypass.off.other Unlimited Ammo - Disabled for {0}
+commands.unlimited.on Unlimited na Ammo - Naka-enabled
+commands.unlimited.off Unlimited na Ammo - Naka-disabled
+commands.unlimited.on.other Unlimited na Ammo - Naka-enabled ng {0}
+commands.unlimited.off.other Unlimited na Ammo - Naka-disabled ng {0}
+commands.spraybypass.on Unlimited na Ammo - Naka-enabled
+commands.spraybypass.off Unlimited na Ammo - Naka-disabled
+commands.spraybypass.on.other Unlimited na Ammo - Naka-enabled ng {0}
+commands.spraybypass.off.other Unlimited na Ammo - Naka-disabled ng {0}
 commands.suspect.invalid Invalid suspect info
```
Comment (Markzt) [inline tg.js:965]: 965 Invalid ang inpormasyon ng suspect

File: tg.js [hunk 12]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
 commands.suspect.menu Suspect in menu
```
Comment (Markzt) [inline tg.js:966]: 966 Suspect sa Menu

File: tg.js [hunk 13]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
 commands.suspect.left Suspect has left
```
Comment (Markzt) [inline tg.js:967]: 967 Ang suspect ay umalis

File: tg.js [hunk 14]
```diff
@@ -942,37 +942,37 @@ modes.prophunt.rotation.reset [Prop Hunt] Rotation reset.
...
 commands.suspect.found Suspect Found. Spectating: {0}, Spawned: {1}, Name: {2}
```
Comment (Markzt) [inline tg.js:968]: 968 nahanap ang suspect. spectating: {0}, Spawned: {1}, pangalan: {2}

File: tg.js (hunk 6 above)
Comment (callmedandann) [reply tg.js]: Done

File: tg.js (hunk 5 above)
Comment (callmedandann) [reply tg.js:928]: "Pag-spectate" is quite long.

File: tg.js (hunk 7 above)
Comment (callmedandann) [reply tg.js]: Done

File: tg.js (hunk 3 above)
Comment (callmedandann) [reply tg.js]: Done

File: tg.js (hunk 4 above)
Comment (callmedandann) [reply tg.js]: Done

**Comment by CaptainJay4201** [comment]
--Reviewing Markzt's Suggestions--
Main errors: Capitalization, spelling
926 - Capitalize "pumili" 
927 - Fix Capitalization on "Pwede"
928 - Revert (panonood is fine as-is in my opinion)
933 - Fix typo "and" to "ang"
935 - Remove random capitalization in second sentence, add period.
965 - Change to "Walang bisa ang impormasyon ng pinaghihinalaan" (This might be a bit long)
966 - Change to "Pinaghihinalaan sa menu" 
967 & 968 - Change "suspect" to "pinaghihinalaan" 
968 - Capitalize "nahanap" and change "spectating" to "Nanonood"

**Review by CaptainJay4201 — COMMENTED** [review]
--New translations, review would be appreciated--
937 - Wala ka ng palitan ng props
938 - Ikaw ay may {0} natitirang palitan 
939 - Rotation May Bisa
940 - Rotation Walang Bisa
941 - Rotation ay na reset.
942 - Login para pwedeng gamitin ang command na ito (Please shorten)
943 - {0} karugtong, {1} nasa menu
944 - Si {0} ay ang current na host.
Possibly translate "rotation" and try to get rid of the extra English as I could not think of a term to use.

**Comment by Markzt** [comment]
979 - translate: {0} Ay Na-ban
980 - translate: {0} Ay Na-Vote Kick
981 - translate: {0} Ay Naaresto ng K.P.D

Change request:
957-964
translate The Ammo to Bala

**Review by Dreamcatcher7D — CHANGES_REQUESTED** [review]
923- Possibly change it to "ay naka-enable" instead, since "naka" already indicates it's happening, we could also just remove "naka" and just make it "ay enabled".
932 - Possibly replace "para makapag spawn" to "upang maka-spawn" or some other similar variant, but "upang maka-spawn" is much shorter.
933 - Possibly replace "laban na ito" to "labang ito"
928 - Possibly replace it "panonood" to "pag-nood"
937- Replace "ng" with "nang", ng is incorrect. 
937 - add "na" before "natitirang" so for example.. "Ikaw ay may (5) na natitirang palitan".
975- "Fino-forfeit" feels icky, possibly change it to "Iniiwan" - translates to "Leaving/Forfeiting/Quitting".

**Comment by Jaehnvnct** [comment]
924 - Suggestion only. Change 'Naka-join' to 'Nakasali' na sa ranked match. Ang pag-abandona ay mag-reresulta ng temporary ban.
932 - Para sa'kin hindi bagay yung 'klas'. Replace 'klas' to 'kategorya', 'makapaglaro to makapag-spawn' Result: "Pumili ng isang kategorya para makapaglaro." 
934 - Hindi na maipapanalo ang round na ito. Change 'winnable' to 'maipapanalo'
933 - Change the current translation to this : "Panalo kayo/ka sa round na ito."

**Comment by callmedandann** [comment]
Clarification: The changes for lines 957-964 are suggested by Markzt.

**Review by Dreamcatcher7D — CHANGES_REQUESTED** [review]
970 - Replace with it "Hintayin munang magsimula ang laro upang maka-forfeit"
971 - Replace it with "Ulitin nalang muli mamaya"
974 - Replace it with "Iniwan na ang laro"
976-977 - Add "si" before the username and "ay" after the username, so.. ex. "si Dreamcatcher ay sumali sa laro".
987 - "napatalsik" feels a bit weird, "ay natanggal sa laro" or "ay tinanggal sa laro" or simply just "ay na-kick" fits better.

**Review by Dreamcatcher7D — CHANGES_REQUESTED** [review]
923 - Add "ang" before "2x"
965 - Replace it with "Walang bisa na inpormasyon ang suspect"
966 - Replace it with "ang suspect ay nasa Menu"

**Review by Navsy17 — DISMISSED** [review]
It's alright.

**Review by callmedandann — COMMENTED** [review]
Ready to merge!

**Pull request merged**